ui/app/AppLayouts/Profile/popups/ChangePasswordModal.qml

```diff
@@ -86,7 +85,7 @@ StatusModal {
         StatusButton {
             id: submitBtn
             objectName: "changePasswordModalSubmitButton"
-            text: qsTr("Change Password")
+            text: qsTr("Change password and restart Status")
```

Comment:
Don't we need to use `&` instead of `and` here?

Reply:
Accordingly to the design - no